onnxscript/function_libs/torch_aten/ops/nn.py

```diff
@@ -239,17 +239,26 @@ def aten_conv_depthwise3d(
     raise NotImplementedError()
 
 
-def aten_cross_entropy_loss(
-    self: TensorType,
-    target: TensorType,
-    weight: Optional[TensorType] = None,
-    reduction: int = 1,
-    ignore_index: INT64 = -100,
+@torch_op("aten::cross_entropy", trace_only=True)
+def aten_cross_entropy(
+    self: TFloatOrBFloat16,
+    target: Sequence[int],
+    weight: Optional[TFloatOrBFloat16] = None,
+    reduction: str = "mean",
```

Comment:
Still suggest to keep the original signature where reduction is an integer. It's very possible that the aten op cross_entropy will receive an integer as the value of 'reduction'.

We might rename this function to _aten_cross_entropy() and keep it as is. Then add a new aten_cross_entropy() function who can accept 'reduction' as an integer.

Reply:
We can just set it to int I think. No need for a new function.

Reply:
I cannot find any document for cross_entropy_loss(), but only cross_entropy(). and in the document about cross_entropy(), the 'reduction' parameter is str, not int.
from my understanding, reduction(int) signature is the original API definition.  while reduction(str) is a helper function which user always want to use.

Reply:
Again we don't worry about what the torch document says (which is a documentation of torch op, not aten ops.). The aten signature is the only source of truth we care about. If it says int, we should use int. This is because torch will internally convert user input into what the then op expects.